ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectLoginTest.kt

```diff
@@ -223,23 +201,18 @@ class RedirectLoginTest : StringSpec() {
             Log.debugWithSupplier {
                 "Redirect Incomplete AuthnRequest In URL Query Test"
             }
+            val authnRequest = createDefaultAuthnRequest()
+            val encodedRequest = encodeAuthnRequest(authnRequest)
             val queryParams = SimpleSign().signUriString(
                     SAML_REQUEST,
-                    createValidAuthnRequest(),
+                    encodedRequest,
                     null)
+            // using !! here because null is already checked with safe call
```

Comment:
✏️ Outside the scope of your changes, but would you mind changing this to
```kotlin
            val qpSamlReq = queryParams[SAML_REQUEST]
            queryParams.set(SAML_REQUEST, qpSamlReq?.substring(0, qpSamlReq.length / 2))
```
The `!!` makes my eyes water.